mypyc/test-data/run-integers.test

```diff
@@ -157,6 +157,15 @@ def check_bitwise(x: int, y: int) -> None:
             check_or(ll, rr)
             check_xor(ll, rr)
 
+[case testIsinstanceNotBoolAndIsInt]
+def test_isinstance_not_bool_and_is_int(value: object) -> bool:
+    return not isinstance(value, bool) and isinstance(value, int)
```

Comment:
Actually this already seems to work correctly. Can you switch the checks so that it will reproduce the incorrect old behavior? I.e., like this:
```py
return isinstance(value, int) and not isinstance(value, bool)
```